### ISSUE TITLE
.net standard version

### ISSUE THREAD

**Issue by boriscallens** [Robmaister/SharpNav#70]
Is there any interest in having a .net standard version of this?

I needed it for use in the Xenko framework and done the work here https://github.com/boriscallens/SharpNav

**Comment by Robmaister** [comment]
Yeah, definitely! Life has kept me super busy, but I've been meaning to be a more active maintainer and keep up with the new .NET Core and .NET Standard stuff.

**Comment by feliwir** [comment]
Any news on this?
